pkgs/by-name/sp/spoofdpi/package.nix

```diff
@@ -0,0 +1,29 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  buildGoModule,
+  ...
+}:
+
+buildGoModule rec {
+  pname = "SpoofDPI";
+  version = "0.11.1";
+
+  src = fetchFromGitHub {
+    owner = "xvzc";
+    repo = "SpoofDPI";
+    rev = "v${version}";
+    hash = "sha256-GdGOnaIDy7XWWo0MOu+HfQcLrW/PDlRxf0y1jjJrZNQ=";
+  };
+
+  vendorHash = "sha256-47Gt5SI6VXq4+1T0LxFvQoYNk+JqTt3DonDXLfmFBzw=";
+
+  meta = {
+    homepage = "https://github.com/xvzc/SpoofDPI";
+    description = "A simple and fast anti-censorship tool written in Go";
+    license = lib.licenses.asl20;
+    platforms = lib.platforms.all;
```

Comment:
Unset this, this is set by buildGoModule AFAIK

Reply:
i've took some pakcages written on go like `gat` and saw standart meta. :D

Reply:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/go/module.nix

eh

Reply:
It's actually this

https://github.com/NixOS/nixpkgs/blob/91b1ab3465a334c2de084e6945238e88c2d1a302/pkgs/build-support/go/module.nix#L314

Reply:
> It's actually this
> 
> https://github.com/NixOS/nixpkgs/blob/91b1ab3465a334c2de084e6945238e88c2d1a302/pkgs/build-support/go/module.nix#L314

oh, ok!